### PR TITLE
Add `isCorsEnabled` option

### PR DESCRIPTION
**Pull request by ffflorian**
For testing purposes it would be nice to be able to disable CORS.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -15,6 +15,13 @@ declare namespace electronServe {
 		*/
 		scheme?: string;
 
+		/**
+		If CORS should be enabled. See https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
+
+		@default `true`
+		*/
+		corsEnabled?: boolean;
```
Comment (sindresorhus) [inline index.d.ts]:
```suggestion
		isCorsEnabled?: boolean;
```

File: index.d.ts [hunk 2]
```diff
@@ -15,6 +15,13 @@ declare namespace electronServe {
 		*/
 		scheme?: string;
 
+		/**
+		If CORS should be enabled. See https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
+
+		@default `true`
```
Comment (sindresorhus) [inline index.d.ts]:
```suggestion
		@default true
```

File: index.d.ts [hunk 3]
```diff
@@ -15,6 +15,13 @@ declare namespace electronServe {
 		*/
 		scheme?: string;
 
+		/**
+		If CORS should be enabled. See https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
```
Comment (sindresorhus) [inline index.d.ts]:
```suggestion
		Whether [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) should be enabled.
```

Comment (sindresorhus) [reply index.d.ts]: And mention the use-case.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
You also need to update the readme.

**Comment by ffflorian** [comment]
@sindresorhus Thanks for the suggestions, I applied them.

**Pull request merged**